torch/onnx/symbolic_opset11.py

```diff
@@ -0,0 +1,22 @@
+from torch.onnx.symbolic_helper import parse_args, _unimplemented
+from torch.onnx.symbolic_helper import _black_list_in_opset
+
+
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+# This file exports ONNX ops for opset 11
+
+black_listed_operators = [
+    "loop", "cumsum", "round", "bitshift", "equal"
```

Comment:
can you add ne as well? (it uses equal)
Make sure to add the symbolic names (I think the symbolic name for equal is "eq").

Reply:
Thanks. There are no symbolics for bitshift, round, and loop

Reply:
in that case you can remove them and keep cumsum, eq and ne